src/components/addTodoForm/index.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react"
+import styles from "components/addTodoForm/form.module.css"
+import trash from "assets/trash.png"
+import { useDispatch } from "react-redux"
+import { addNewTodo, isAdding } from "redux/actions"
+import { customSanitize } from "utils/sanitizeInput"
+
+function AddTodoForm() {
+  const dispatch = useDispatch()
+  const [task, setTask] = useState("")
```

Comment:
See if we can set this initial state as null or not. 
```suggestion
  const [task, setTask] = useState(null)
```

Reply:
bhaia when we keep null as a default value it throws an error saying default value can’t be null use empty string instead